docs/usage/self-hosted-configuration.md

```diff
@@ -328,6 +328,14 @@ For example, if you set `dockerChildPrefix=myprefix_` then the final container c
 !!! note
     Dangling containers are only removed when Renovate runs again with the same prefix.
 
+## dockerCliOptions
+
+Adds command-line docker CLI options to Renovate sidecar Docker containers.
+
+For example, `--memory=1g --cpus="1.5"`.
+
+Read the [Docker runtime resource constraints](https://docs.docker.com/config/containers/resource_constraints/) for more information.
```

Comment:
It feels like we're missing an explanation on our side here. How about a structure like this:

```markdown
One line summary of config option.

Explanation about _why_ and _when_ a user may need this config option.
Expand on the `--memory=1g --cpus="1,5"` stuff.

Real config example in proper JSON.

To learn more about ... read the Docker runtime resource constrains.
```

Reply:
updated